compiler/src/codegen/optimize_mod.re

```diff
@@ -43,9 +43,7 @@ let default_global_optimization_pre_passes =
 let default_function_optimization_passes =
     (~optimize_level, ~shrink_level, wasm_mod) => {
   List.concat([
-    // All the additions here are optional if DWARF must be preserved. That is,
-    // when DWARF is relevant we run fewer optimizations.
-    // FIXME: support DWARF in all of them.
+    // TODO(#1500): Preserve DWARF debug info for all optimizations
```

Comment:
This is a FIXME for binaryen; I'd restore the deleted comment but delete the FIXME (and just close the issue)